observability/WFLY-19846-promote-mp-tel-2-to-default.adoc

```diff
@@ -0,0 +1,115 @@
+---
+categories:
+- microprofile
+- observability
+stability-level: default
+issue: https://github.com/wildfly/wildfly-proposals/issues/619
+feature-team:
+ developer: jasondlee
+ sme:
+  -
+ outside-perspective:
+  -
+promotes: https://issues.redhat.com/browse/WFLY-19590
+promoted-by: https://github.com/wildfly/wildfly-proposals/issues/619
+---
+= [Default] Promote MicroProfile Telemetry 2.0 to WildFly Standard
+:author:            Jason Lee
+:email:             jasondlee@redhat.com
+:toc:               left
+:icons:             font
+:idprefix:
+:idseparator:       -
+
+== Overview
+
+MicroProfile Telemetry 2.0 shipped with WildFly Preview 34. To meet WildFly and product roadmap goals, this feature needs to be migrated to the WildFly feature pack, and its stability level promoted to `DEFAULT`.
+
+=== User Stories
+
+User wants to deploy an application using MicroProfile Telemetry 2.0 from MicroProfile Platform 7.0.
+
+== Issue Metadata
+
+- https://issues.redhat.com/browse/WFLY-19846[]
+
+=== Related Issues
+
+- https://issues.redhat.com/browse/EAP7-2227[]
+
+=== Affected Projects or Components
+
+- MicroProfile Fault Tolerance
+- AMQP
+- Kafka
+
+=== Other Interested Projects
+
+=== Relevant Installation Types
+
+* Traditional standalone server (unzipped or provisioned by Galleon)
+* OpenShift Source-to-Image (S2I)
+* Bootable jar
+
+== Requirements
+
+* WildFly must fully support MicroProfile Telemetry 2.0:
+** Tracing - Existing support
+** Metrics - New feature
+** Logs - New feature
+* OpenTelemetry versions must be updated to match those required by MicroProfile Telemetry 2.0
+
+=== Changed requirements
+
+N/A
+
+=== Non-Requirements
+
+Currently, neither OpenTelemetry nor MicroProfile Telemetry are supported in domain mode. While this is now technically feasible, given the push nature of the export process, support for this will have to be delivered under a different (to-be-file) feature request.
+
+=== Future Work
+
+N/A
+
+== Backwards Compatibility
+
+* There are no known breaking changes. The upstream OpenTelemetry project made a number of breaking changes to the metrics semantic conventions (which could affect user dashboards), but metrics support will be new with this release, so those changes will not affect existing installations.
```

Comment:
MP Telemetry main page explicitelly mentioned two backward incompatible changes against MP telemetry 1.1 that we used earlier. Can you please double-check this?

https://microprofile.io/specifications/telemetry/2-0/

Reply:
Ah, right. I was thinking API changes, but that semantic convention change is worth noting. I don't think the env var is of too much interest to WF directly. I'll update the language here.